Client/Script/BulkShiftOverlay.js

```diff
@@ -63,6 +63,11 @@ class BulkShiftOverlay {
         this.#mediaItem = mediaItem;
     }
 
+    /** 
+     * Keeps track of markers without existing rows, so we can add them to the ignored markers table later
+     * @type {Array} */
+    #markersWithoutRows = [];
```

Comment:
I think it'd be cleaner if the [`shift` command](https://github.com/danrahn/IntroEditorForPlex/blob/332e8cc6a4f0c539e44bc667a5d120d0de7f9de5/Client/Script/Common.js#L92C5-L92C10) was instead modified to have a [`MarkerEnum` parameter](https://github.com/danrahn/IntroEditorForPlex/blob/332e8cc6a4f0c539e44bc667a5d120d0de7f9de5/Shared/MarkerType.js#L23) indicating which marker types the shift should apply to.

Reply:
Thanks for the response, I agree that does sound a lot cleaner, that's a bit beyond my skills though.

I'll have to leave it to you then, I figured you would have a better solution, it being your project lol, I just wanted to throw this up just in case.

Reply:
I took a stab at this as part of 23f01f1. It includes a bunch of extra stuff that's not directly related to filtering by marker type, so it ended up far more complicated than your changes, but it should accomplish the same thing.

Reply:
Looks good! Thanks for adding it, very helpful for anime and other shows that have recaps of last episodes and previews for next episodes.